scripts/generate/templates/SqlShim.mustache

```diff
@@ -0,0 +1,41 @@
+{-|
+Module: Database.Persist.Sql.Shim
+
+Defines all the @persistent@ functions lifted into 'MonadSqlQuery', and
+re-exports a version of the @Database.Persist.Sql@ module replacing each
+function with the lifted version.
+
+This file is autogenerated, to keep it in sync with
+@Database.Persist.Monad.Shim@.
+-}
+
+{- THIS FILE IS AUTOGENERATED AND SHOULD NOT BE EDITED MANUALLY -}
+
+{-# LANGUAGE CPP #-}
+
+module Database.Persist.Sql.Shim (
+  module Database.Persist.Sql,
+  {{#functions}}
```

Comment:
why not re export the module?

Reply:
The trouble with me forgetting about this PR for six months: this is a question [you already asked](https://github.com/brandonchinn178/persistent-mtl/pull/47#discussion_r835509264). Listing the functions here avoids exporting `fromAcquire` from the shim.